packages/lab/src/stepped-tracker/TrackerStep/TrackerStep.tsx

```diff
@@ -19,46 +21,27 @@ import trackerStepCss from "./TrackerStep.css";
 
 const withBaseName = makePrefixer("saltTrackerStep");
 
-type State = "default" | "completed";
-
-type StateWithActive = State | "active";
+type IconOptions = "default" | "completed" | "warning" | "error";
 
 export interface TrackerStepProps extends ComponentPropsWithoutRef<"li"> {
   /**
+   * The slug of the icon to be displayed in the TrackerStep
+   */
+  icon?: IconOptions | null;
+  /**
+   * @deprecated Use `icon` prop instead
```

Comment:
Why are we deprecating props in lab?

Reply:
When I asked about process around renaming props, @mark-tate suggested deprecation. Definitely cleaner to just remove IMO, if that's allowed for labs?

Reply:
labs is allowed to make breaking changes, deprecation is only needed in core components